cmd/describe/cluster/cmd.go

```diff
@@ -353,6 +353,13 @@ func run(cmd *cobra.Command, argv []string) {
 		}
 	}
 
+	str = fmt.Sprintf("%s"+
+		"Audit Log Forwarding:       %s\n", str, getAuditLogForwardingStatus(cluster))
```

Comment:
This is only for HCP right? Should we check the cluster type prior to showing these field? It's just that current classic customers maybe inquire how to do it and there isn't a way at the moment

Reply:
that is a great idea!!!
Will make it appear only for hcp.